src/autotype/AutoType.cpp

```diff
@@ -423,6 +425,11 @@ void AutoType::performGlobalAutoType(const QList<QSharedPointer<Database>>& dbLi
         return;
     }
 
+    // Invalidate last match if it's old enough
+    if (m_lastMatch.first != nullptr && QDateTime::currentMSecsSinceEpoch() - m_lastMatchTime > 30000) {
```

Comment:
should be: `if (m_lastMatch.first && (QDateTime::currentMSecsSinceEpoch() - m_lastMatchTime) > 30000) {`

Reply:
As you please.